packages/color-slider/test/color-slider.test.ts

```diff
@@ -448,6 +450,26 @@ describe('ColorSlider', () => {
 
         expect(el.sliderHandlePosition).to.equal(46.875);
     });
+    it('accepts input events', async () => {
```

Comment:
To ensure that we've appropriately addressed the originating issue, can you add a `changeSpy` style test, a la: https://github.com/adobe/spectrum-web-components/blob/1ac12931771c6d5fdbc99f5d214702ed644cb81a/packages/sidenav/test/sidenav.test.ts#L182 

Each keyboard interaction with this element should dispatch one `input` and one `change` event. In the case that you make a protracted `pointer` event, it should dispatch `input` event(s) for each change in value until `pointerup` at which point a `change` event should be dispatched.